mint/longFormatDataSeries.ts

```diff
@@ -1,24 +1,43 @@
 import {computed} from "@vue/composition-api";
 import {LongFormatSeriesMetadata, SeriesDefinition, SeriesMetadata, WideFormatSeriesMetadata} from "./fakeAPIData";
 import {FilteringProps, useFiltering} from "./filteredData";
+import {Dictionary} from "vuex";
+import {useTransformation} from "./tranformedData";
+import {def} from "@vue/composition-api/dist/utils";
 
 interface Props extends FilteringProps {
     series: SeriesDefinition[]
     metadata: SeriesMetadata
 }
 
 export function useLongFormatData(props: Props) {
-    const {filteredData} = useFiltering(props)
+    let settings = props.settings;
+    if (props.metadata.settings) {
+        // filter to those settings specified in the metadata
+        settings = props.metadata.settings
```

Comment:
Since we have a lot of settings that aren't used for filering, but just for cost calculations, figured it was more efficient to restrict the settings here to keep array iterations small when actually filtering.